packages/deployments/contracts/contracts_forge/core/connext/helpers/Executor.t.sol

```diff
@@ -203,19 +239,190 @@ contract ExecutorTest is ForgeHelper {
         data
       )
     );
+  }
+
+  // Should gracefully handle failures if value != amount
```

Comment:
On top of the previous comment about the edge case where value != amount, and amount == 0, would like to see that case tested here